_data/himanshu004/himanshu004.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Himanshu Madan
+institution: Deen Dayal Updhyaya College
+profile_pic: me2.jpg 
+quote: Falling into the river does not drowns you, not willing to swim does.
```

Comment:
Fix grammatical error? `drowns` to `drown`.

Reply:
Corrected the error.